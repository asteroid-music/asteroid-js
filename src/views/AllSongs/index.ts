//Main component
export { AsteroidSongList } from './AllSongs';


