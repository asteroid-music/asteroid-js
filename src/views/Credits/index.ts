//Main component
export { AsteroidCredits } from './Credits';


