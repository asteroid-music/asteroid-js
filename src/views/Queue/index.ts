//Main component
export { AsteroidQueue } from './Queue';


