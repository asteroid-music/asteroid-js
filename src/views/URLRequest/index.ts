//Main component
export { AsteroidURLRequester } from './URLRequest';


