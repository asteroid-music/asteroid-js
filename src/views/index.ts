export { AsteroidCredits } from './Credits';
export { AsteroidQueue } from './Queue';
export { AsteroidURLRequester } from './URLRequest';
export { AsteroidSongList } from './AllSongs';


