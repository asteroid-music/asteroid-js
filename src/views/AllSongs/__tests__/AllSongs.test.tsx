import React from 'react';
import renderer from 'react-test-renderer';
import { AsteroidSongList as SongList } from 'views/AllSongs';

describe('<SongList />',()=>{
    //Basic rendering with minimal props
    describe('basic rendering',()=>{
        it("renders correctly",()=>{
            expect(renderer
                .create(<SongList/>)
                .toJSON()
            ).toMatchSnapshot();
        });
    });
});