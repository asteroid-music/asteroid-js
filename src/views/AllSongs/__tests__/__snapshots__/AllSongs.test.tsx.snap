// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SongList /> basic rendering renders correctly 1`] = `
<Material UI Container>
  <div>
    <Material UI Typography>
      Requesting queue data... 
    </Material UI Typography>
    <Material UI CircularProgress />
  </div>
</Material UI Container>
`;
